src/components/posts/conclusion.css

```diff
@@ -0,0 +1,57 @@
+.horizontal {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+}
+
+.card {
+    background-color: #f0f0f0;
+    border-radius: 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 620px;
+    height: 42.5rem;
+    padding: 1.25rem;
+}
+
+.head {
+    font-weight: bold;
+}
```

Comment:
This can probably be abstracted to a utility class like `.text-bold` - does this already exist in our codebase?